Makefile

```diff
@@ -37,15 +53,15 @@ deploy: generate-deploy
 deploy-test: generate-test
 	$(KUBECTL) apply -f config/test/kubemacpool.yaml
 
-generate-deploy: manifests
-	kustomize build config/release > config/release/kubemacpool.yaml
+generate-deploy: $(KUSTOMIZE) manifests
```

Comment:
regarding `deploy:` here [above](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/134/commits/2da21b8d68ac67b71292831313dc5180c52b8be4#diff-b67911656ef5d18c4ae36cb6741b7965R50): shouldn't it apply `config/release/kubemacpool.yaml` instead of `config/test/kubemacpool.yaml`?

Reply:
We deploy test/kubemacpol.yaml because its for development environment, and in this yaml we pass 
''debug'' flag to the logs so we get everything for debugging